app/src/services/cpk.ts

```diff
@@ -313,6 +384,45 @@ class CPKService {
           to: collateral.address,
           value: marketData.funding,
         })
+      } else if (useCompoundReserve && compoundTokenDetails) {
+        if (userInputCollateral.address === pseudoNativeAssetAddress) {
```

Comment:
we need a hardcoded gas value for native asset transactions, it looks like market creation with compound integration costs around 1,300,000 gas so a hardcode of around 1,500,000 gas would be great!

Reply:
Added gas value hardcode